lib/wallaby/query.ex

```diff
@@ -29,6 +29,7 @@ defmodule Wallaby.Query do
     * `:count` - The number of elements that should be found (default: 1).
     * `:visible` - Determines if the query should return only visible elements (default: true).
     * `:text` - Text that should be found inside the element (default: nil).
+    * `:get` - The number of the element to return if multimple elements satisfy the selection criteria.
```

Comment:
Hm, I guess I like `choose` and `pick` better, `get` seems a wee bit to generic for me, might also be because getting a specific radio button is called `choose` in capybara but that might also get confusing...